sssd_test_framework/roles/ipa.py

```
@@ -811,6 +1040,150 @@ def __get_member_args(self, members: list[IPAUser | IPAGroup | str]) -> list[str
+        :type gid: str | None, defaults to None.
+        :return: self.
+        :rtype: IDGroupOverride
+        """
+
+        attrs: CLIBuilderArgs = {
+            "desc": (self.cli.option.VALUE, description),
+            "group-name": (self.cli.option.VALUE, name),
+            "gid": (self.cli.option.VALUE, gid),
+        }
+
+        self.role.host.conn.exec(["ipa", "idoverridegroup-add", idview_name, self.name] + list(self.cli.args(attrs)))
+        return self
+
+    def modify_override(
+        self,
+        idview_name: str,
+        *,
+        description: str | None = None,
+        name: str | None = None,
+        gid: int | None = None,
+    ) -> IDGroupOverride:
+        """
+        Modify an Group ID override.
+
+        :param idview_name: Name of IDView.
+        :type idview_name: str
+        :param description: Description.
+        :type description: str | None, defaults to None
+        :param name: Overridden group name.
+        :type name: str | None, defaults to None
+        :param gid: Overridden Group ID Number.
+        :type gid: str | None, defaults to None
+        :return: self.
+        :rtype: IDGroupOverride
+        """
+
+        attrs: CLIBuilderArgs = {
+            "desc": (self.cli.option.VALUE, description),
+            "group-name": (self.cli.option.VALUE, name),
+            "gid": (self.cli.option.VALUE, gid),
+        }
+        attrs = CLIBuilderArgs(attrs)
+        self.role.host.conn.exec(["ipa", "idoverridegroup-mod", idview_name, self.name] + list(self.cli.args(attrs)))
+
+        return self
+
+    def delete_override(self, idview_name: str) -> IDGroupOverride:
+        """
+        Delete an Group ID override.
+        :param idview_name: Name of IDView.
+        :type idview_name: str
+        :return: self.
+        :rtype: IDGroupOverride
+        """
+
+        self.role.host.conn.exec(["ipa", "idoverridegroup-del", "--continue", idview_name, self.name])
```

Was there a specific reason to add `--continue` here?

---

Continuous mode: Don't stop on errors.

---

Yes I understand what it is for, but in general I think it's better to return an error instead of ignoring it.